src/registry/router.js

```diff
@@ -7,7 +7,7 @@ const ComponentRoute = require('./routes/component');
 const ComponentsRoute = require('./routes/components');
 const ComponentInfoRoute = require('./routes/component-info');
 const ComponentPreviewRoute = require('./routes/component-preview');
-const ListComponentsRoute = require('./routes/list-components');
+const IndexRoute = require('./routes/index');
```

Comment:
The UI's "home page" used to be called "list components" route but now it's becoming much more than that. So list components => index